website/public/css/custom-asciinema-player.css

```diff
@@ -38,15 +38,15 @@
 
 @media screen and (min-width: 640px) {
```

Comment:
I know this media query is taken from old code. Since we have a set of definite `breakpoints` for the website, we should utilize that here.